.github/workflows/ci.yaml

```diff
@@ -314,32 +314,27 @@ jobs:
             # cache node modules using the same key as restore.
           key: ${{ steps.node_modules-cache-restore.outputs.cache-primary-key }}
 
-  Starchart-Docker:
+  Docker-Build-Deploy:
     # We'll only build and push when landing on main, not for PRs
     if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
     runs-on: ubuntu-latest
     # Don't bother unless everything else passes
     needs: [Build, Prettier, Unit-Tests, E2E-Tests, Dockerfile-Lint, ESLint, Type-Check]
+    env:
+      IMAGE: ghcr.io/DevelopingSpace/starchart
     permissions:
       contents: read
       packages: write
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      - name: Define GITHUB_SHORT_SHA
+        run: echo "GITHUB_SHA_SHORT=$(echo $GITHUB_SHA | cut -c 1-6)" >> $GITHUB_ENV
```

Comment:
I don't think this really makes a difference but aren't the characters on the tail end of a sha the most meaningful?

Reply:
You can take any hash and truncate it to use only the first part of it as a short-hand.  This is what GitHub does when they show my commit as `93a5088` vs. `93a508896294bdd9f74b1d4ca92a57c49999f34e`